crates/ra_ide_api/src/extend_selection.rs

```diff
@@ -134,8 +136,8 @@ fn pick_best<'a>(l: SyntaxToken<'a>, r: SyntaxToken<'a>) -> SyntaxToken<'a> {
     return if priority(r) > priority(l) { r } else { l };
     fn priority(n: SyntaxToken) -> usize {
         match n.kind() {
-            WHITESPACE => 0,
-            IDENT | SELF_KW | SUPER_KW | CRATE_KW | LIFETIME => 2,
+            tkn![' '] => 0,
```

Comment:
For whitespace, I think we should just use the name WHITESPACE, withot macro

Reply:
Ok.